streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -182,10 +152,54 @@ public String toString(final String indent) {
         return sb.toString();
     }
 
+    protected Map<TopicPartition, Long> recordCollectorOffsets() {
+        return Collections.emptyMap();
+    }
+
+    protected void updateOffsetLimits() {
+        log.trace("{} Updating store offset limits", logPrefix);
```

Comment:
Just trying to be consistent with other task-level log entries. We can also just remove this as you mentioned below.